apps/rush-lib/src/cli/actions/VersionAction.ts

```diff
@@ -227,7 +227,7 @@ export class VersionAction extends BaseRushAction {
     });
 
     if (changeLogUpdated) {
-      git.addChanges('.', this.rushConfiguration.changesFolder);
+      git.addChanges(this.rushConfiguration.changesFolder, '.');
```

Comment:
This is the default, so you can just cut the last param off here

Reply:
Actually, this one is still correct. We are trying to add every change in the folder. Could you revert this change?

Reply:
Sorry, too tired 🙈

Mixed it up here and thought it's the same issue as below.